managing_providers/_topics/automation_management_providers.md

```diff
@@ -103,17 +103,13 @@ your playbooks.
 
 4.  Add a description for the repository in the **Description** field.
 
-5.  Select an **SCM Type** from the drop-down menu.
+5.  Add a **URL** or IP Address for the repository.
 
-6.  Add a **URL** or IP Address for the repository.
+6.  Select the appropriate **SCM Credentials** from the drop-down menu.
 
-7.  Select the appropriate **SCM Credentials** from the drop-down menu.
+7.  Provide a branch name in the **SCM Branch** field.
 
-8.  Provide a branch name in the **SCM Branch** field.
-
-9.  Check the appropriate box for any **SCM Update Options**.
-
-10. Click **Add**.
+9. Click **Save**.
```

Comment:
Think this should be 8?

Reply:
Alslo probably should have 2 spaces after the `.` to align the content

Reply:
Updated the changes.